libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFOAuthTestFlowCoordinatorDelegate.m

```diff
@@ -89,4 +89,12 @@ - (BOOL)oauthCoordinatorIsNetworkAvailable:(SFOAuthCoordinator*)coordinator {
     return self.isNetworkAvailable;
 }
 
+- (void)oauthCoordinator:(SFOAuthCoordinator *)coordinator didBeginAuthenticationWithSafariViewController:(SFSafariViewController *)svc {
+    [self log:SFLogLevelDebug format:@"%@ called.", NSStringFromSelector(_cmd)];
```

Comment:
Could we actually follow the web-based exception throwing as the delegate methods above do?  It'd be good to sanity check those, in case they're inadvertently called.